draft-group-privacypass-k-check.md

```diff
@@ -238,25 +244,45 @@ cache-control: max-age=3600
 <Bytes containing the target's BHTTP-encoded response>
 ~~~
 
-# K-Check
-
-Clients are configured with the URLs for one or more mirror resources. Each URL identifies an API
-endpoint that clients use to obtain mirrored copies of a resource.
+# Using Mirrors for Consistency Checks {#integration}
 
-The input to K-Check is a candidate HTTP resource, a target URL at which the resource
-was obtained, and a representation of the input resource. To check this
-resource, the client runs the following steps for each configured mirror.
+Clients can use mirrors to implement consistency checks for a candidate HTTP
+resource. In particular, in possession of the target URL at which the resource
+was obtained, as well as an authoritative representation of the resource, clients
+can check to see if this resource is consistent with that of the mirror's as follows:
 
 1. Send a mirror request to the mirror for the target URL. If the request fails, fail
-   this mirror check.
+   this consistency check.
 1. Otherwise, compute the first valid representation of the resource based on the mirror's response.
-1. Compare the computed representation to the input representation. If they do not match,
-   fail this mirror check. Otherwise, this mirror check succeeds.
+1. Compare the computed representation to the input resource representation. If they do not match,
+   fail this consistency check. Otherwise, this consistency check succeeds.
+
+The benefits of using the mirror protocol to check consistency depend on a multitude of
+factors, including, but not limited to, the number of clients interacting with a particular
+mirror, whether or not the mirror is trustworthy, and application requirements for dealing
+with consistency check failures.
+
+<!-- TODO: weave in considerations here -->
```

Comment:
Which considerations? Is this a place to link to failure considerations in a later section?

Reply:
It could be a link to the failure considerations or they could be inlined here, yeah. Whatever reads better.

Reply:
Got it